raster/r.path/main.c

```diff
@@ -219,13 +219,13 @@ int main(int argc, char **argv)
     if (G_parser(argc, argv))
         exit(EXIT_FAILURE);
 
-    strcpy(dir_name, opt.dir->answer);
+    G_strlcpy(dir_name, opt.dir->answer, sizeof(dir_name));
```

Comment:
`G_strlcpy()` may truncate the source string if destination buffer is too small. How about to check this and `G_fatal_error()` out if that is the case?